src/lib/wallet/GoodWalletClass.js

```diff
@@ -212,9 +212,10 @@ export class GoodWallet {
    * @returns {Promise<R>|Promise<R|*>|Promise<*>}
    */
   async listenTxUpdates(fromBlock: int = 0, blockIntervalCallback: Function) {
-    log.debug('listenTxUpdates listening from block:', fromBlock)
     const curBlock = await this.wallet.eth.getBlockNumber()
-    fromBlock = new BN(fromBlock <= curBlock ? fromBlock : curBlock)
+    const dayAgoBlock = Math.max(0, fromBlock - 17280)
```

Comment:
what is 17280?

Reply:
24hourss seconds divided by 5 seconds blocktime

Reply:
can u move it to constant and write a comment what is it?